src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
Cannot say it's mistake but splitting lines when it's not necessary make code harder to read

Reply:
I understand and will follow the advice in the future, but in this case there is no way to make changes except to change the name for the constant or use delimiters because the string becomes 102 characters, if you think it is worth changing the name of the constant to improve the readability of the code, I will do it